Kernel/Syscalls/execve.cpp

```diff
@@ -792,9 +792,13 @@ KResultOr<RefPtr<FileDescription>> Process::find_elf_interpreter_for_executable(
     if (main_program_header.e_type == ET_DYN) {
         // If it's ET_DYN with no PT_INTERP, then it's a dynamic executable responsible
         // for its own relocation (i.e. it's /usr/lib/Loader.so)
-        if (path != "/usr/lib/Loader.so")
-            dbgln("exec({}): WARNING - Dynamic ELF executable without a PT_INTERP header, and isn't /usr/lib/Loader.so", path);
-        return nullptr;
+        if (path != "/usr/lib/Loader.so") {
+            dbgln("exec(={}): WARNING - Dynamic ELF executable without a PT_INTERP header, and isn't /usr/lib/Loader.so", path);
```

Comment:
I think it would be good to make these `dmesgln(..)` instead of `dbgln(..)` so they are visible to the user inside the OS, if they run `dmesg`. 
```suggestion
            dmesgln("exec(={}): WARNING - Dynamic ELF executable without a PT_INTERP header, and isn't /usr/lib/Loader.so", path);
```